docker/Dockerfile.gnu8_mpich314_cuda101

```diff
@@ -20,7 +20,7 @@ RUN wget -q http://www.mpich.org/static/downloads/3.1.4/mpich-3.1.4.tar.gz && \
 
 RUN git clone -b v8.0.0b3 --depth 1 https://github.com/cupy/cupy.git && \
     cd cupy && pip3 install . && cd .. && \
-    git clone https://bitbucket.org/mpi4py/mpi4py.git && \
+    git clone -b 3.0.3 https://bitbucket.org/mpi4py/mpi4py.git && \
```

Comment:
git cloning from bitbucket!?!?

RIP mercurial...

Reply:
it subversion or nothing, dude